integration/tests/access/unstaked_node_test.go

```diff
@@ -123,6 +123,9 @@ func (suite *UnstakedAccessSuite) buildNetworkConfig() {
 		testnet.WithLogLevel(zerolog.TraceLevel),
 	)
 
+	acsConfig := testnet.NewNodeConfig(flow.RoleAccess)
```

Comment:
is there a need for another access node? Can't we just use the `stakedConfig` as the Secure Access node, it too will open port 9001.

Reply:
Yup your right